sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/BlobItem.java

```diff
@@ -236,48 +231,25 @@ public BlobItem setCurrentVersion(Boolean isCurrentVersion) {
     }
 
     /**
-     * Get the objectReplicationPolicyId property: The
-     * objectReplicationPolicyId property.
-     *
-     * @return the objectReplicationPolicyId value.
-     */
-    public String getObjectReplicationPolicyId() {
```

Comment:
These APIs were added in the recent beta release, so this should not be a breaking change